assets/css/themes/_dev.scss

```diff
@@ -1,5 +1,5 @@
 body.dev-theme {
-  *:not(.ad-btn):not(.ad-btn-image) {
+  *:not(.ad-btn):not(.ad-btn-image):not(.ignore-theme) {
```

Comment:
Nice, can you add ignore-theme to the ad-btn and ad-btn-image elements and remove them here?